packages/components/src/utils/inject-styles.js

```diff
@@ -17,15 +17,17 @@ limitations under the License.
 const { BorderRadius, BoxShadow, Container, Display, Hovers, Position, Skins, Spacing } = require('@orion-ui/style/lib/2016-12-01');
 
 (function injectStyles() {
-  if (!document.body) { return; }
+  if (!document) { return; }
+  if (!document.currentScript) { return; }
```

Comment:
Can you do a console.error here as well? I understand this might be here so we can run the tests in Node? But in the browser we'll want to report an error since styles are critical.